### PR TITLE
Add new StandupBot to remind people about standups

### PR DESCRIPTION
**Pull request by jacksonj04**
Inspired by parlbot, this is a bot which will look at the [standups page on the Wiki](https://wiki.mysociety.org/wiki/Standups), figure out which ones to tell people about, and then send a channel-wide reminder.

### CODE REVIEW

File: wiki/bin/standupbot.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+# Handy flags to help with debugging...
+# Don't actually push messages to Slack, just show the message payload locally
+DO_NOT_SEND = False
+# Add all standups to the announce list, regardless of time
+ANNOUNCE_ALL = False
+# Announce in a specific channel, rather than the one listed in the Wiki
+# False is a default value, but '@yourusername' and '#sandbox' are also good
+ANNOUNCE_IN = False
+
+print 'StandupBot'
```
Comment (davea) [inline wiki/bin/standupbot.py]: Fly-by 'Python 3 support please' moan 😃

Comment (jacksonj04) [inline wiki/bin/standupbot.py]: I'm behind on modern Pythonicisms, what's not Python 3 friendly?

Comment (dracos) [inline wiki/bin/standupbot.py]: print() is a function (so on py2 you'll need to `from __future__ import print_function`).
There is a `2to3` script python comes with that gives you some hints, but doesn't maintain 2-compatibility. The only issues I can see are print(), and you'll need to handle the differing handling of urllib2/urllib.

**Comment by jacksonj04** [comment]
Hopefully should now be Python 2/3 friendly (or at least it appears to be so whilst testing locally)

File: wiki/bin/standupbot.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    print('Today is a weekday, starting the process!')
+
+    site = MediaWiki(user_agent='StandupBot')
+
+    # Regex patterns
+    listPattern = re.compile('\* (.*)')
+    sectionPattern = re.compile('<!-- Begin Standups Schedule -->(.*?)<!-- End Standups Schedule -->', re.M | re.S)
+    standupPattern = re.compile('\|-\n\| (.*?)\n\| (.*?)\n\| (.*?)\n\| (.*?)\n', re.M | re.S)
+
+    announceForTime = datetime.datetime.now() + datetime.timedelta(minutes=1)
```
Comment (davea) [inline wiki/bin/standupbot.py]: IMO this would be better if it announced standups at the time they start, as opposed to a minute early.

**Pull request merged**